pg/index.d.ts

```diff
@@ -63,41 +61,34 @@ export interface ResultBuilder extends QueryResult {
 }
 
 export declare class Pool extends events.EventEmitter {
-
-    constructor();
-
     // `new Pool('pg://user@localhost/mydb')` is not allowed.
     // But it passes type check because of issue:
     // https://github.com/Microsoft/TypeScript/issues/7485
-    constructor(config: PoolConfig);
+    constructor(config?: PoolConfig);
 
     connect(): Promise<Client>;
     connect(callback: (err: Error, client: Client, done: () => void) => void): void;
 
     end(): Promise<void>;
 
-    query(queryText: string): Promise<QueryResult>;
-    query(queryText: string, values: any[]): Promise<QueryResult>;
+    query(queryText: string, values?: any[]): Promise<QueryResult>;
 
     query(queryText: string, callback: (err: Error, result: QueryResult) => void): void;
     query(queryText: string, values: any[], callback: (err: Error, result: QueryResult) => void): void;
 
-    public on(event: "error", listener: (err: Error, client: Client) => void): this;
-    public on(event: "connect", listener: (client: Client) => void): this;
-    public on(event: "acquire", listener: (client: Client) => void): this;
-    public on(event: string, listener: Function): this;
```

Comment:
It looks like the most general overload (args are `string` and `Function` resp) is dropped throughout. Does the `on` method support event/listeners outside those specified in the other overloads?